Track Your Reps/TabBarController.swift

```diff
@@ -25,11 +28,10 @@ class TabBarController: UITabBarController, UITabBarControllerDelegate {
     
     func setupUser() {
         guard let userState = userState else { onboardUser(); return }
+        User.sharedInstance.delegate = self
         User.sharedInstance.state = userState
         User.sharedInstance.fetchMembers()
-        User.sharedInstance.fetchEvents() {
-            self.setupViewControllers()
```

Comment:
so no closure anymore to setup VCs, we'll do this via the SetupViewControllersDelegate